spm/private/spm_xcode_toolchain.bzl

```diff
@@ -138,10 +138,10 @@ The location for the utilities that are required by SPM.\
 """,
         ),
         "_swift_worker": attr.label(
-            cfg = "host",
+            cfg = "exec",
             allow_files = True,
             default = Label(
-                "@build_bazel_rules_swift//tools/worker",
+                "@build_bazel_rules_swift//tools/worker:worker_wrapper",
```

Comment:
Why was this change needed?

Reply:
Not sure that it was. I noticed that `rules_swift` was using it. The target is an `alias` that resolves to `//tools/worker:worker` or `//tools/worker:universal_worker`.